src/test/regress/sql/multi_colocation_utils.sql

```diff
@@ -57,11 +57,6 @@ CREATE FUNCTION shards_colocated(bigint, bigint)
     AS 'citus'
     LANGUAGE C STRICT;
 
-CREATE FUNCTION get_colocated_table_array(regclass)
```

Comment:
It already exists in colocation_utils tests. Before this change, since we do not need this function in SQL scope and we need to test it, we created this UDF just for testing. With this change we now need it is SQL scope also. Therefore I removed this UDF creation from here and add it to the upgrade sql.